content/contributing/development/ui/scss.rst

```diff
@@ -0,0 +1,256 @@
+
+.. _contributing/scss:
+
+===============
+SCSS Guidelines
+===============
+
+
+
+Syntax and Formatting
+=====================
+
+.. tabs::
+
+    .. code-tab:: html SCSS
```

Comment:
I can for this example, but the following ones needs variables.
I switched to `html` for "visual consistency" within the page, otherwise it would look off, no?